mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogBinaryReader.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace Mono.Profiler.Log {
+
+	sealed class LogBinaryReader : BinaryReader {
+
+		public LogBinaryReader (LogStream stream, bool leaveOpen)
+			: base (stream, LogReader.Encoding, leaveOpen)
+		{
+		}
+
+		public new void FillBuffer (int count)
```

Comment:
What's the point of introducing this method?

Reply:
`LogProcessor` needed public access to it, and the base method is `protected`. But it doesn't matter; this approach didn't work anyway. I've pushed a new approach.